src/ui/markdown.tsx

```diff
@@ -164,6 +182,8 @@ export function RawLit({
             else if (dimensions && dimensions.length == 1)
                 return <img alt={value.path} src={resourcePath} width={dimensions[0]} />;
             else return <img alt={value.path} src={resourcePath} />;
+        } else if(value.embed) {
+            return <Embed linkText={value.fileName()} embedderPath={sourcePath} inline={inline} />
```

Comment:
Embeds also work for sections and named blocks, so I would check for the block type (heading and block) and append that to the linktext if relevant.